src/fault/Fault.cpp

```diff
@@ -205,11 +256,40 @@ static int32_t uniqueFaultId()
     static int32_t cid = -1;
     return cid--;
 }
-Fault::Fault()
+
+Fault::Fault() : id_(uniqueFaultId())
 {
     etiss::log(etiss::VERBOSE, std::string("etiss::fault::Fault::Fault() called. "));
-    id_ = uniqueFaultId();
 }
+
+Fault::Fault(int nullid) : id_(nullid) {}
+
+Fault::Fault(const Fault &cpy)
```

Comment:
these copy/move constr/assign are equivalent to the default

Reply:
Yes. Was not aware of the main CMakeLists.txt already forcing c++14
https://github.com/tum-ei-eda/etiss/pull/100/commits/eff549bed670e9bc0968e4fcddd43791e0fdacd6